include/pros/imu.h

```diff
@@ -57,6 +57,10 @@ typedef struct __attribute__((__packed__)) euler_s {
 } euler_s_t;
 
 #define IMU_MINIMUM_DATA_RATE 5
+#define IMU_EULER_LIMIT 180
+#define IMU_HEADING_MAX 360
+
+#define DEGTORAD (M_PI / 180)
```

Comment:
unless these are used in this header file somewhere, they should probably get moved to the source file. avoid adding internal values/utilities to the public interface wherever possible

Reply:
(and yes DEGTORAD may be useful for users but in the interests of separation of concern it should at the very least not be defined in this file, if it's to be given to our users at all)
(which i think it shouldn't be, especially because this is the kind of symbol commonly defined by users anyway and to define it here now might cause conflicts)